src/bin/sol-fbp-runner/Makefile

```diff
@@ -3,4 +3,11 @@ bin-sol-fbp-runner-$(FBP_RUNNER) := main.c runner.c
 
 ifeq (y,$(INSPECTOR))
 bin-sol-fbp-runner-$(FBP_RUNNER) += inspector.c
+
+ifeq (y,$(HTTP_SERVER))
```

Comment:
it's nested under `ifeq (y,$(INSPECTOR))` that is baaed on the same config value that results in `SOL_FLOW_INSPECTOR_ENABLED`